consent-service-impl/src/main/java/io/mosip/esignet/services/ConsentServiceImpl.java

```diff
@@ -86,4 +86,12 @@ public ConsentDetail saveUserConsent(UserConsent userConsent) {
                 AuditHelper.buildAuditDto(userConsent.getClientId()), null);
         return consentDetailDto;
     }
+
+    @Override
+    @Transactional
+    public void deleteUserConsent(String clientId, String psuToken) {
+        if(consentRepository.existsByClientIdAndPsuToken(clientId,psuToken)) {
+            consentRepository.deleteByClientIdAndPsuToken(clientId, psuToken);
```

Comment:
why is it required to check the existence and then delete? We can directly trigger delete r8